cms/publications/tests.py

```diff
@@ -4,6 +4,47 @@
 from cms.publications.models import Publication, PublicationIndexPage, TOC
 
 
+class TestDocumentContainer(TestCase):
+    fixtures = ["fixtures/document_container.json"]
+
+    def setUp(self):
+        response = self.client.get("/pub-index-page/publication-with-attachments/")
+        soup = BeautifulSoup(response.content, "html.parser")
+        self.external_url, self.internal_page, self.pdf, self.docx, self.unknown = [
+            str(attachment) for attachment in soup.select(".attachment")
+        ]
+
+    def test_has_bytes(self):
+        self.assertIn("2\xa0bytes", self.pdf)
+
+    def test_has_title_and_summary(self):
+        self.assertIn("3a pdf 3a", self.pdf)
+        self.assertIn("3b Summary: pdf 3b", self.pdf)
+
+    def test_pdf(self):
+        self.assertIn("pdf.svg", self.pdf)
+        self.assertIn("<abbr", self.pdf)
+        self.assertIn("Portable", self.pdf)
+
+    def test_word(self):
+        self.assertIn("doc.svg", self.docx)
+        self.assertIn("MS Word Document", self.docx)
+
+    def test_unknown(self):
+        self.assertIn("file.svg", self.unknown)
+        self.assertIn(">File<", self.unknown)
+
+    def test_internal_page(self):
+        self.assertIn("link.svg", self.internal_page)  # is this correct behaviour?
```

Comment:
I think this _is_ correct behaviour, but we should discourage this kind of thing in the operations manual.